### ISSUE TITLE
Other Libraries

### ISSUE THREAD

**Issue by Positive07** [rm-code/love-api#4]
LÖVE provides a whole set of libraries with it:

- [ ] [Luasocket](https://www.love2d.org/wiki/socket)
- [ ] [Lua-ENET](https://www.love2d.org/wiki/lua-enet)
- [ ] [UTF-8](https://www.love2d.org/wiki/utf8)
- [ ] [BitOps](http://bitop.luajit.org/)
- [ ] [LuaFFI](http://luajit.org/ext_ffi.html)
- [ ] [LuaJIT](http://luajit.org/ext_jit.html) (This one is not really important, there are not many things you can do with it anyway)

It would be nice to provide access to this libraries in the same format, Is this in the scope of this project? If this is fine, I can write a few of them (UTF and Network libs mostly, since I have used them many times)

**Comment by rm-code** [comment]
Yeah I was thinking about adding them as well, but I am not sure how to include them in the current doc. 

Maybe they could be put into a separate file for 3rd party libraries?

**Comment by Positive07** [comment]
Separate file seems better to me.

In a directory called `3rd-party` sounds good too

**Comment by Positive07** [comment]
Slime stated that Lua BitOps, LuaJIT and LuaFFI are not really part of LÖVE but of LuaJIT which LÖVE is built against, so they are not really official supported library, but is safe to assume they will be present since LÖVE is built against LuaJIT more of the time and there are no plans to go to standard Lua
